lib/policy/policyValidator.js

```diff
@@ -0,0 +1,106 @@
+'use strict'; // eslint-disable-line strict
+
+const Ajv = require('ajv');
+const userPolicySchema = require('./userPolicySchema');
+const errors = require('../errors');
+
+const ajValidate = new Ajv({ allErrors: true });
+// compiles schema to functions and caches them for all cases
+const userPolicyValidate = ajValidate.compile(userPolicySchema);
+
+// parse ajv errors and build list of erros
+function _parseErrors(ajvErrors) {
+    let parsedErr;
+    ajvErrors.some(err => {
+        const resource = err.dataPath.replace('.', '');
+        if (err.keyword === 'required' && err.params) {
+            const field = err.params.missingProperty;
+            if (field === 'Version') {
+                parsedErr = errors.MissingPolicyVersion;
+            } else if (field === 'Statement') {
+                parsedErr = errors.MissingPolicyStatement;
+            } else if (field === 'Action') {
+                parsedErr = errors.MissingPolicyAction;
+            } else if (field === 'Effect') {
+                parsedErr = errors.MissingPolicyEffect;
+            } else if (field === 'Resource') {
+                parsedErr = errors.MissingPolicyResource;
+            } else {
+                parsedErr = errors.InvalidPolicyDocument;
+            }
+        } else if (err.keyword === 'minItems' && resource === 'Statement') {
```

Comment:
I didn't understand your comment, I need resource to evaluate the else if block and it changes with every err item in the list. Also, all it's doing is changing `.Statement` => `Statement`